test/expiration_worker_test.rb

```diff
@@ -17,10 +17,10 @@ def test_one_time_code_expiration
     new_valid_one_time_code
     expire_and_assert(encryption: 2, diagnosis: 1)
 
-    move_forward_seconds(10 * 60 - 2) # T+00:09:58
+    move_forward_seconds(1440 * 60 - 2) # T+23:59:58
```

Comment:
Could do `move_forward_days(1) ; move_forward_seconds(-2)` probably and then have the database state be consistent without adjusting the helper.

Reply:
Awesome - changed!